tests/test_console.py

```diff
@@ -87,6 +87,10 @@ def test_errors_if_provided_non_numeric_input(console_with_invalid_io_input):
     assert "Input was not a number!" in str(err.value)
 
 
+def test_sends_message_to_console_io(console_with_valid_io_input):
+    console_io = console_with_valid_io_input.get_console_io()
```

Comment:
It's good that you're using fixtures to set up tests and to hide irrelevant details from the user.

In general, a test should contain no more and no less information that is needed for someone to see the inputs to the system under test and expected outputs. Abstracting the setup should only hide irrelevant details. Sometimes, a setup method/function that accepts some parameters to describe the initial state might be a better choice than a fixture.

I'm struggling to figure out exactly what is being tested here and in `test_accepts_an_integer_from_the_user`.

If you can, revisit these and see if a fixture, a helper function or no hiding of detail would provide greater clarity.